jekyll/_cci2/configuration-reference.md

```diff
@@ -313,17 +314,17 @@ Template | Description
 {% raw %}`{{ epoch }}`{% endraw %} | the current time in seconds since the unix epoch.
 {: class="table table-striped"}
 
-During step execution template will be replaced by runtime values and resulted string will be cache key.
+During step execution, the templates above will be replaced by runtime values and using the resultant string as the `key`.
 
 Template examples:
  * {% raw %}`myapp-{{ checksum "package.json" }}`{% endraw %} - cache will be regenerated every time something is changed in `package.json` file, different branches of this project will generate the same cache key.
  * {% raw %}`myapp-{{ .Branch }}-{{ checksum "package.json" }}`{% endraw %} - same as the previous one, but each branch will generate separate cache
  * {% raw %}`myapp-{{ epoch }}`{% endraw %} - every build will generate separate cache
 
-While choosing suitable templates for you cache `key` keep in mind that cache saving is not a free operation, because it will take some time to upload cache to our storage. So it make sense to have a `key` that generates new cache only if something actually changed and avoid generating a new one every build.
+While choosing suitable templates for your cache `key`, keep in mind that cache saving is not a free operation, because it will take some time to upload the cache to our storage. So it make sense to have a `key` that generates new cache only if something actually changed and avoid generating a new one every build.
```

Comment:
"...it makes sense to have a `key` that generates *a* new cache only if something actually changed *to* avoid generating a new one every build."